README.md

```diff
@@ -3,106 +3,214 @@
 [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/catberry/main?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=body_badge)
 
-[![NPM](https://nodei.co/npm/catberry.png)](https://nodei.co/npm/catberry/)
-
 <p align="center">
   <img src="https://raw.githubusercontent.com/catberry/catberry/master/docs/images/logo.png" />
 </p>
 
-Catberry was developed to help in creating ["isomorphic" Web applications](https://github.com/catberry/catberry/blob/7.1.2/docs/index.md#isomorphic-applications).
-Isomorphic applications are apps that use the same codebase to run from both
-server and client-side environments.
-
-This means you write the code only once, and deploy it the way you want.
-Catberry handles all the differences between these two environments.
-
-You will get
-[Single Page Application](http://en.wikipedia.org/wiki/Single-page_application)
-using browser's
-[History API](https://developer.mozilla.org/en-US/docs/Web/Guide/API/DOM/Manipulating_the_browser_history).
-And back-end that renders exactly the same page on the server.
-
-Here you can find full
-[Catberry Documentation](https://github.com/catberry/catberry/blob/7.1.2/docs/index.md).
-
-To get started with an example, please proceed to
-[Get Started Guide](https://github.com/catberry/catberry/blob/7.1.2/docs/index.md#get-started).
-
-If you want to try TodoMVC application, proceed to [this link](https://github.com/catberry/catberry-todomvc).
-
-## Advantages
-* [Flux](https://github.com/catberry/catberry/blob/7.1.2/docs/index.md#flux) architecture
-* [Cat-components](https://github.com/catberry/catberry/blob/7.1.2/docs/index.md#cat-components) (like [web-components by Google](http://webcomponents.org/) but isomorphic)
-* You do not need to write any code for registration or definition of
-[cat-components](https://github.com/catberry/catberry/blob/7.1.2/docs/index.md#cat-components) or
-[stores](https://github.com/catberry/catberry/blob/7.1.2/docs/index.md#stores),
-Catberry discovers them automatically on startup
-* Fast and efficient server-side [progressive rendering engine](http://www.phpied.com/progressive-rendering-via-multiple-flushes/) based on
-[node.js streams](http://nodejs.org/api/stream.html#stream_api_for_stream_implementors)
-* [Dust](https://github.com/catberry/catberry-dust),
-[Handlebars](https://github.com/catberry/catberry-handlebars) and
-[Jade](https://github.com/catberry/catberry-jade) template engines
-* No Virtual DOM – you can use any other library with Catberry without conflicts
+## What the cat is that?
+
+Catberry was developed to help in creating ["isomorphic/Universal" Web applications](https://github.com/catberry/catberry/blob/8.0.0-dev/docs/index.md#isomorphicuniversal-applications).
```

Comment:
Catberry was developed to help create ["isomorphic/Universal" Web applications](https://github.com/catberry/catberry/blob/8.0.0-dev/docs/index.md#isomorphicuniversal-applications).